esp/platform/espp.hpp

```diff
@@ -28,6 +28,13 @@ typedef ISocket * isockp;
 typedef MapBetween<int, int, isockp, isockp> SocketPortMap;
 typedef CopyReferenceArrayOf<ISocket> SocketPortArray;
 
+class CEspElevatedLoggingFilter : public CategoryLogMsgFilter
+{
+    public:
+        CEspElevatedLoggingFilter() : CategoryLogMsgFilter(MSGAUD_all, MSGCLS_elevated, DebugMsgThreshold, false) {}
+        bool mayIncludeCategory(const LogMsgCategory & cat) const { return (cat.queryAudience() & audienceMask) && (cat.queryClass() & classMask) && (cat.queryDetail() <= maxDetail); }
```

Comment:
I'm still not sure why this doesn't just return true as long as the class matches?

Reply:
return cat.queryClass() & classMask;

Reply:
Remove the audience and threshold tests